app/backend/templates/v2/activeness_probe.txt

```diff
@@ -0,0 +1,12 @@
+Hi {{ user.name }}!
+
+It's been a while since you logged in to Couchers.org, and we just wanted to check if you are still actively hosting surfers?
```

Comment:
```suggestion
We noticed you haven't logged in for a while and wanted to check if you're still open to hosting surfers. To keep your "Can Host" status, please log back in within the next 15 days. Otherwise, we'll change your hosting status to "Can't Host."
```